src/stdlib/models/slicecategories.jl

```diff
@@ -9,13 +9,15 @@ using StructEquality
   hom::HomT
 end
 
-@struct_hash_equal struct SliceC{ObT, HomT, C<:Model{Tuple{ObT, HomT}}} <: Model{Tuple{SliceOb{ObT, HomT}, HomT}}
+@struct_hash_equal struct SliceC{ObT, HomT, C}
```

Comment:
I like this massive reduction in type constraining code

Reply:
Given the considerations in my comments below, it seems like this would become 

```julia 
@struct_hash_equal struct SliceC{ObT, HomT, C<:Model} <: Model
  ... # same code inside
end
```
which is still an improvement!